Doc/whatsnew/3.14.rst

```diff
@@ -106,6 +106,15 @@ Removed
   by Jelle Zijlstra in :gh:`118767`.)
 
 
+pathlib
+-------
```

Comment:
Please can you move this above the "Others" section I added in https://github.com/python/cpython/pull/118799? We can list the modules alphabetically like in 3.13.